airflow/serialization/serializers/pandas.py

```diff
@@ -17,6 +17,7 @@
 # under the License.
 from __future__ import annotations
 
+from io import BytesIO
```

Comment:
This changed lazy import to eager import, which in serde code we like to keep to a minimum